### ISSUE TITLE
Impossible to activate Drum's Starpower

### ISSUE THREAD

**Issue by arielenter** [fofix/fofix#47]
Hello again xD

I'm still trying fofix 4 alpha 2, and I found that is impossible to activate the drum's starpower. Usually when the drums is about to use a starpower the is a free style part where the player is suppose to hit the drums several times as he please before hitting the last note (the 4th one) to activate the star power. The player is suppose to hit the drums many times to make the last note appears, and while he does the notes rails get bigger and bigger, but fofix 4 alpha 2 this doesn't happens. Neither the rails get bigger and the last note never appears :(

This would be fine if it wouldn't for the fact that every time you fail to use the starpower the song give you another chance over and over through all song not allowing you to play the song.

I try to look for an option to disable star power all along but such option doesn't seem to exist. Is there a work around to this for now?

Thank you :)

**Comment by mdsitton** [comment]
Yeah there are quite a few bugs in the drums atm. It just doesn't get as much attention by me, because it can be a pain to lug out the drumset haha.

So fyi recently I was in the process of cleaning up the hit detection code for fofix, maybe when I get back to that I can see if this is still an issue. 

There is plenty of starpower bugs overall as well, sometimes you will randomly get full bar of sp sometimes sp phrases don't get marked missed if you miss notes within it etc. I plan to get those looked at when i finish up the cleanup of the input/hit detection code... (its a mess)

We are wanting to have some kind of release out before January, even if its another alpha release.

Thank you for the bug reports :)

**Comment by mdsitton** [comment]
So I'm fairly sure this is still an issue in master.

**Comment by arielenter** [comment]
I try using the newest master on my GNU/Linux, but I'm getting the following error:
NameError: global name 'configSplit' is not defined

And if I delete previous fofix.ini I get the following:
IOError: [Errno 2] No such file or directory: '/home/ariel/.fofix/fofix.ini'

Maybe there is a dependency that I haven't installed? or maybe something that is not be compile? I didn't have this problem with the master I downloaded last week which is still working.

**Comment by arielenter** [comment]
I'm sorry I should have publish the entire error. It was this:
File "/home/ariel/Escritorio/fofix-master/fofix/core/Config.py", line 70, in read
    configLines = [line for line in configSplit
NameError: global name 'configSplit' is not defined

What I did is I open up Config.py and looked for "configSplit". And I notice that the programmer should probably wanted to meant "configLines" and not "configSplit", so I made the rectification and it worked :)

Sadly the problem still persist and is still impossible to activate drum's starpower.

**Comment by mdsitton** [comment]
Yeah I already fixed the issue before I even saw your replies here. I actually fixed that issue before I pushed  this morning, but it didn't make it into the commit haha.

**Comment by arielenter** [comment]
I saw you put a startpower button on the mapping section of the Drums. Thanks :)

Is not the solution but meanwhile it will help xD

Thanks a lot :+1:
